Sources/API/Core/XML/xml_token.h

```diff
@@ -44,9 +44,8 @@ class XMLToken
 /// \{
 
 public:
-	XMLToken() : type(NULL_TOKEN), variant(SINGLE)
-	{
-	}
+	XMLToken() : type(NULL_TOKEN), variant(SINGLE), name(), value(), attributes()
```

Comment:
Not sure why you'd want to explicitly call the std::string and std::vector default constructors here as they are always being called.

To clean up a constructor like that one the best way would probably be to completely remove the constructor, and then use "type = NULL_TOKEN;" down where the member variable is declared.